functions.php

```diff
@@ -114,14 +114,14 @@ function create_post_type_team()
         'menu_icon' => 'dashicons-megaphone',
         'has_archive' => true,
         'supports' => array(
-            'title'
+            'title',
+            'thumbnail'
         ),
         'can_export' => true,
         'taxonomies' => array(
             'post_tag',
             'category'
-        ),
-        'show_in_rest' => true
```

Comment:
This was probably an accident, but please add `show_in_rest` back in! Same for the other post types.